scripts/devnet/spammer.py

```diff
@@ -33,11 +33,17 @@ def __init__(self, name: str, listen_port: int,
                  metrics: Optional[dict] = None,
                  container_image: Optional[str] = None):
         self.address = "NQ40 GCAA U3UX 8BKD GUN0 PG3T 17HA 4X5H TXVE"
+
+        if topology_settings.spammer_profile is None:
```

Comment:
Could you maybe add a `getter`? I don't recommend to access a class attribute directly